routingcatdog/src/Components/CatRandom.js

```diff
@@ -0,0 +1,34 @@
+import React, { useState, useEffect } from 'react';
+import axios from "axios";
+import Dog from "./Dog"
+
+
+const CatRandom =()=> {
+  
+    const [imgLink,setImgLink] = useState(null)
```

Comment:
Since you know youre going to get a string of information, this might be useful to have an empty string as the initial value, so to anyone reading this would know what its expecting